### PR TITLE
LPS-168310 Redirect patterns: disable save btn when there is an error

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
See https://github.com/brianchandotcom/liferay-portal/pull/126761#issuecomment-1325640884

Screenshot 2022-11-24 at 15 11 54

Only the last commit needs review. The rest have been already approved

Screenshot 2022-11-24 at 15 06 08

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
Hey @adolfopa @marcogalluzzi I'm not sure what is the best absolute urls example: 

1. "http://www." + PropsValues.COMPANY_DEFAULT_WEB_ID " => http://www.liferay.com
2. PortalUtil.getPortalURL(_httpServletRequest)

I finally choosed the second one, buut if other is prefered, feel free to change

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

File: modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectPatternConfigurationDisplayContext.java [hunk 1]
```diff
@@ -74,6 +75,14 @@ public Map<String, Object> getRedirectPatterns() {
 			}
 		).put(
 			"portletNamespace", _liferayPortletResponse.getNamespace()
+		).put(
+			"strings",
+			HashMapBuilder.put(
+				"absoluteUrl", PortalUtil.getPortalURL(_httpServletRequest)
+			).put(
+				"relativeUrl",
+				PropsUtil.get("default.guest.public.layout.friendly.url")
```
Comment (adolfopa) [inline modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectPatternConfigurationDisplayContext.java:84]: You need to use `PropsValues.DEFAULT_GUEST_...`

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Review by marcogalluzzi — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
Fixed https://github.com/liferay-lima/liferay-portal/pull/3770#discussion_r1032082658. Going to forward it.

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**